### DISCUSSION TITLE
Cannot flash other models another using the classification exemple

### DISCUSSION THREAD

**Discussion by bruno-darochac** [bitcraze/discussions#551] [Q&A]
Hello, I've tried to remplace in the MakeFile of the classification exemple your classification tflite by an [other](https://tfhub.dev/iree/lite-model/ssd_mobilenet_v1_100_320/uint8/default/1), to see what append. I succeed on the make build command but the flash stuck at 99%. Have you any idea or suggestion on what I can try ? 

Thanks in advance

**Comment by krichardsson** [comment]
There used to be a bug that caused this symptom in older versions of the the AI deck software. It is important to upgrade the ESP firmware and GAP8 bootloader to the latest version as described in [the getting started guide](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/). Try to [reflash the GAP8 bootloader](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader-for-the-aideck-1-0-or-aideck-1-1-rev-a-c-) to make sure you actually have the latest version.

**Comment by krichardsson** [comment]
Please also see #369 and #484
